docs/_includes/common/filter-menu.html

```diff
@@ -1,8 +1,8 @@
 <div id="filter-menu">
     <h2><span class="secondary-text">@ebay/skin/</span>filter-menu</h2>
 
-    <h3 id="filter-menu">Filter Menu – ARIA Version – Default</h3>
-    <p>The ARIA version of the filter menu is a <em>statelful</em> menu. It's primary purpose is used in conjunction with <a href="#filter-menu-button">filter-menu-button</a>,
+    <h3 id="filter-menu">Filter Menu – Default</h3>
+    <p>The default version of the filter menu is a <em>statelful</em> menu. It's primary purpose is used in conjunction with <a href="#filter-menu-button">filter-menu-button</a>,
```

Comment:
`statelful`

Reply:
Wish this were a suggestion and then I could just take the suggestion. 😉 

Will fix.